index.js

```diff
@@ -120,6 +120,19 @@ function updateLink() {
     window.history.replaceState(null, document.title, link);
 }
 
+/**
+ * Gets the index of the asset folder
+ * @param {string} folderName The name of the asset folder
+ * @returns number or null if the folder name could not be found
+ */
+function getFolderIndexFromName(folderId) {
```

Comment:
Name -> Id?

Reply:
@bghgary yeah, this should be Id instead of Name.  Just made the update.